.github/workflows/test_with_clone.yaml

```diff
@@ -0,0 +1,275 @@
+              shell: bash
+              run: |
+                # The <CLONE>_PROXY_ADMIN account role will be used to manage interschema objects (e.g. tasks, dynamic tables)
+                # within the cloned database. First let's create the <CLONE>_PROXY_ADMIN account role name and export for immediate use:
+                echo "SNOWFLAKE_CLONE_PROXY_ADMIN_ROLE=${SNOWFLAKE_DOMAIN_CLONE_SANITIZED}_PROXY_ADMIN" >> $GITHUB_ENV
+
+            - name: Create account role PROXY_ADMIN
+              shell: bash
+              run: |
+                # Briefly switch roles and create the <CLONE>_PROXY_ADMIN role
+                snow sql -q "USE ROLE USERADMIN; CREATE OR REPLACE ROLE ${SNOWFLAKE_CLONE_PROXY_ADMIN_ROLE};"
+
+
+            ########################################################################################################
+            ########################################################################################################
+            ########################################## MANAGING GRANTS #############################################
+            ########################################################################################################
+            ########################################################################################################
+
+            - name: Transfer ownership of current and future tasks and dynamic tables to PROXY_ADMIN
+              shell: bash
+              run: |
+                snow sql -q "USE ROLE SECURITYADMIN; GRANT OWNERSHIP ON ALL DYNAMIC TABLES IN SCHEMA ${SNOWFLAKE_DOMAIN_CLONE_SANITIZED}.SYNAPSE TO ROLE ${SNOWFLAKE_CLONE_PROXY_ADMIN_ROLE} COPY CURRENT GRANTS;"
+                snow sql -q "USE ROLE SECURITYADMIN; GRANT OWNERSHIP ON ALL TASKS IN SCHEMA ${SNOWFLAKE_DOMAIN_CLONE_SANITIZED}.SYNAPSE_RAW TO ROLE ${SNOWFLAKE_CLONE_PROXY_ADMIN_ROLE} COPY CURRENT GRANTS;"
+                snow sql -q "USE ROLE SECURITYADMIN; GRANT OWNERSHIP ON ALL DYNAMIC TABLES IN SCHEMA ${SNOWFLAKE_DOMAIN_CLONE_SANITIZED}.SCHEMACHANGE TO ROLE ${SNOWFLAKE_CLONE_PROXY_ADMIN_ROLE} COPY CURRENT GRANTS;"
+                
+                # We need to revoke OWNERSHIP on future objects from ${SNOWFLAKE_PROXY_ADMIN_ROLE} first, because only one role can have the OWNERSHIP privilege on an object.
+                # There's no "transferring of ownership" involved because these objects don't exist yet.
+                snow sql -q "USE ROLE SECURITYADMIN; REVOKE OWNERSHIP ON FUTURE DYNAMIC TABLES IN SCHEMA ${SNOWFLAKE_DOMAIN_CLONE_SANITIZED}.SYNAPSE FROM ROLE ${SNOWFLAKE_PROXY_ADMIN_ROLE};"
+                snow sql -q "USE ROLE SECURITYADMIN; REVOKE OWNERSHIP ON FUTURE TASKS IN SCHEMA ${SNOWFLAKE_DOMAIN_CLONE_SANITIZED}.SYNAPSE_RAW FROM ROLE ${SNOWFLAKE_PROXY_ADMIN_ROLE};"
+                snow sql -q "USE ROLE SECURITYADMIN; REVOKE OWNERSHIP ON FUTURE DYNAMIC TABLES IN SCHEMA ${SNOWFLAKE_DOMAIN_CLONE_SANITIZED}.SCHEMACHANGE FROM ROLE ${SNOWFLAKE_PROXY_ADMIN_ROLE};"
```

Comment:
Similarly, this line has no effect because there is no OWNERSHIP grant on future dynamic tables in this schema.